documentation/conf.py

```diff
@@ -56,4 +56,4 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = ["_static"]
```

Comment:
Intentional?

Reply:
yes, it was throwing warnings so I removed it.